config.json

```diff
@@ -1412,8 +1420,8 @@
       {
         "groupId": "com.squareup.okio",
         "artifactId": "okio",
-        "version": "2.10.0",
-        "nugetVersion": "2.10.0.3",
+        "version": "3.0.0",
```

Comment:
It looks like this version uses some new packaging format that our tooling does not support.  This `.jar` is empty, and thus the package we are making for it is empty.  The resulting NuGet will not contain C# or Java code and will not be useful for our users.

Let's hold off on updating this package until we can correctly support it.

The actual implementation may have moved to https://mvnrepository.com/artifact/com.squareup.okio/okio-jvm/3.0.0.

Reply:
reverted